pkg/operator/configobservation/cloudprovider/observe_cloudprovider.go

```diff
@@ -9,15 +9,23 @@ import (
 	configv1 "github.com/openshift/api/config/v1"
 	"github.com/openshift/library-go/pkg/operator/configobserver"
 	"github.com/openshift/library-go/pkg/operator/events"
+	"github.com/openshift/library-go/pkg/operator/resourcesynccontroller"
 
 	"github.com/openshift/cluster-kube-controller-manager-operator/pkg/operator/configobservation"
 )
 
-// ObserveCloudProviderNames observes the loud provider from the global cluster infrastructure resource.
+const (
+	targetNamespaceName       = "openshift-kube-controller-manager"
+	cloudProviderConfFilePath = "/etc/kubernetes/static-pod-resources/configmaps/cloud-config/config"
+	configNamespace           = "openshift-config"
```

Comment:
@derekwaynecarr I think we want this to be `openshift-config-managed` to be consistent for all clouds and allow future filtering, right?

Reply:
Okay, if this has to change https://github.com/openshift/installer/pull/1516/files#diff-8db489d59cedb0a4f04c7546779f58a2R68 should be updated to reflect new location right?

Reply:
The documentation of Infrastructure should change, too, then.
https://github.com/openshift/api/blob/d2f01e7b77a6fc78b328db20285423838419fef7/config/v1/types_infrastructure.go#L28